### PR TITLE
Modify help text.

### PR DESCRIPTION
**Pull request by taizhng**
Didn't like it, added clarity and reduced dumb punctuation.

### CODE REVIEW

File: components/sidebar/SearchView/HelpModal.tsx [hunk 1]
```diff
@@ -41,28 +41,27 @@ const HelpModal = ({ isOpen, onClose }: props) => {
...
                 <FormHelperText fontWeight={400}>
-                  For instance,
+                  For example:
                   <UnorderedList>
                     <ListItem>
-                      if you know the specific course code for a first year
+                      If you know the specific code for a first year
                       political science course you need to take, you can search
-                      for {'"pol101"'};
+                      for {'"POL101"'}.
```
Comment (eamonma) [inline components/sidebar/SearchView/HelpModal.tsx:53]: We intentionally do not want to demonstrate the search of capital course codes.

File: components/sidebar/SearchView/HelpModal.tsx [hunk 2]
```diff
@@ -41,28 +41,27 @@ const HelpModal = ({ isOpen, onClose }: props) => {
...
                     </ListItem>
                     <ListItem>
-                      if you know you want to take a course in pseudoscience,
-                      you can search for {'"eco"'};
+                      If you want to take a course in pseudoscience,
+                      you can search for {'"ECO"'}.
```
Comment (eamonma) [inline components/sidebar/SearchView/HelpModal.tsx:57]: As above.

File: components/sidebar/SearchView/HelpModal.tsx [hunk 3]
```diff
@@ -41,28 +41,27 @@ const HelpModal = ({ isOpen, onClose }: props) => {
...
                     <ListItem>
-                      if you know you want to take a course about global
-                      affairs, you can search for {'"human rights"'};
+                      If you want to take a course about global
+                      affairs, you can search for {'"human rights"'}.
                     </ListItem>
                     <ListItem>
-                      if you know you want to take the easiest course at the
-                      university, you can search for {'"mat337"'}
+                      If you want to take the easiest course at UofT, you can search for {'"MAT337"'}
```
Comment (eamonma) [inline components/sidebar/SearchView/HelpModal.tsx:64]: As above.

**Review by eamonma — CHANGES_REQUESTED** [review]

File: components/sidebar/SearchView/HelpModal.tsx (hunk 3 above)
Comment (taizhng) [reply components/sidebar/SearchView/HelpModal.tsx]: I've made the changes below, thank you for your insight.

**Review by eamonma — APPROVED** [review]

**Pull request merged**